### PR TITLE
Line fixes

### PR DESCRIPTION
**Pull request by blair1618**
More nearly matches the line behavior in tangram-js

### CODE REVIEW

**Comment by blair1618** [comment]
@tallytalwar take a look when you get a chance, thanks!

**Comment by tallytalwar** [comment]
Sure, will do near the end of day today.

File: core/src/style/polylineStyle.cpp [hunk 1]
```diff
@@ -84,10 +87,8 @@ bool evalStyleParamWidth(StyleParamKey _key, const DrawRule& _rule, const Tile&
     int zoom  = _tile.getID().z;
     double tileSize = _tile.getProjection()->TileSize();
```
Comment (tallytalwar) [inline core/src/style/polylineStyle.cpp]: `tileSize` redundant now!

File: core/src/style/polylineStyle.cpp [hunk 2]
```diff
@@ -177,12 +178,10 @@ void PolylineStyle::buildLine(const Line& _line, const DrawRule& _rule, const Pr
         } else { height = extrude[1]; }
     }
 
-    GLfloat layer = _props.getNumeric("sort_key") + params.order;
```
Comment (tallytalwar) [inline core/src/style/polylineStyle.cpp]: So not using sort_key values for ordering of roads now?

Comment (blair1618) [inline core/src/style/polylineStyle.cpp]: Nope, tangram-js doesn't use "sort_key" at all unless specified in the
scene file

File: core/src/style/polylineStyle.cpp (hunk 1 above)
Comment (blair1618) [inline core/src/style/polylineStyle.cpp]: Right you are; will remove

**Pull request merged**